Calculator/app/src/androidTest/java/com/sigmasoftwere/akucherenko/calculator/ExampleInstrumentedTest.kt

```diff
@@ -0,0 +1,24 @@
+package com.sigmasoftwere.akucherenko.calculator
```

Comment:
Just a minor comment (typo in the package name): 
"sigmasoftware"

Reply:
Fixid